### DISCUSSION TITLE
dvd75

### DISCUSSION THREAD

**Discussion by dvd75** [community/community#147702] [Other Feature Feedback, Questions, & Ideas]
Wie kan mij zeggen voor wat ik mijn account heb moeten aan maken 3 of 4 jaar geleden dvd75  .?? Was dat voor games ?  Of valuta rekening  in Duitsland die ik toen heb moeten aanmaken maar nooit ben opgeraakt ?Anders is dat heel belangrijk voor mij !!  Laat mij iets weten ik weet het is en rare vraag maar dat licht aan mijn gezondheid

**Comment by mecodeatlas** [comment]
Hi @dvd75, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support.
